mithril-common/src/entities/signed_entity_type.rs

```diff
@@ -356,14 +398,24 @@ mod tests {
             .get_json_beacon()
             .unwrap();
         assert_same_json!("15", &msd_json);
+
+        let cardano_database_full_json =
+            SignedEntityType::CardanoDatabase(CardanoDbBeacon::new(12, 987))
+                .get_json_beacon()
+                .unwrap();
+        assert_same_json!(
+            r#"{"epoch":12,"immutable_file_number":987}"#,
+            &cardano_database_full_json
+        );
     }
 
     // Expected ord:
-    // MithrilStakeDistribution < CardanoStakeDistribution < CardanoImmutableFilesFull < CardanoTransactions
+    // MithrilStakeDistribution < CardanoStakeDistribution < CardanoImmutableFilesFull < CardanoTransactions < CardanoDatabase
```

Comment:
This would be the expected order:
```suggestion
    // MithrilStakeDistribution < CardanoStakeDistribution < CardanoImmutableFilesFull < CardanoDatabase < CardanoTransactions
```

Reply:
Done ✅